deployment/GI03FLMA_Deploy.csv

```diff
@@ -121,37 +121,37 @@ AR30-03,,AR35-05,,GI03FLMA-RIM01-02-CTDMOG044,5,1,2018-06-10T13:59:00,2019-08-11
-AR35-05,,,,GI03FLMA-RIM01-02-CTDMOG042,6,1,2019-08-07T13:34:00,,CGMGI-03FLMA-00006,,CGINS-CTDMOG-11638,59.77010,-39.88419,,60,2692,
-AR35-05,,,,GI03FLMA-RIM01-02-CTDMOG043,6,1,2019-08-07T13:34:00,,CGMGI-03FLMA-00006,,CGINS-CTDMOG-10222,59.77010,-39.88419,,90,2692,
-AR35-05,,,,GI03FLMA-RIM01-02-CTDMOG044,6,1,2019-08-07T13:34:00,,CGMGI-03FLMA-00006,,CGINS-CTDMOG-11595,59.77010,-39.88419,,130,2692,
-AR35-05,,,,GI03FLMA-RIM01-02-CTDMOG045,6,1,2019-08-07T13:34:00,,CGMGI-03FLMA-00006,,CGINS-CTDMOG-10253,59.77010,-39.88419,,180,2692,
-AR35-05,,,,GI03FLMA-RIM01-02-CTDMOG046,6,1,2019-08-07T13:34:00,,CGMGI-03FLMA-00006,,CGINS-CTDMOG-10225,59.77010,-39.88419,,250,2692,
-AR35-05,,,,GI03FLMA-RIM01-02-CTDMOG047,6,1,2019-08-07T13:34:00,,CGMGI-03FLMA-00006,,CGINS-CTDMOG-10223,59.77010,-39.88419,,350,2692,
-AR35-05,,,,GI03FLMA-RIM01-02-CTDMOG048,6,1,2019-08-07T13:34:00,,CGMGI-03FLMA-00006,,CGINS-CTDMOG-10256,59.77010,-39.88419,,500,2692,
-AR35-05,,,,GI03FLMA-RIM01-02-CTDMOH049,6,1,2019-08-07T13:34:00,,CGMGI-03FLMA-00006,,CGINS-CTDMOH-10265,59.77010,-39.88419,,750,2692,
-AR35-05,,,,GI03FLMA-RIM01-02-CTDMOH050,6,1,2019-08-07T13:34:00,,CGMGI-03FLMA-00006,,CGINS-CTDMOH-10227,59.77010,-39.88419,,1000,2692,
-AR35-05,,,,GI03FLMA-RIM01-02-CTDMOH051,6,1,2019-08-07T13:34:00,,CGMGI-03FLMA-00006,,CGINS-CTDMOH-11597,59.77010,-39.88419,,1500,2692,
-AR35-05,,,,GI03FLMA-RIS01-00-SIOENG000,6,1,2019-08-07T13:34:00,,CGMGI-03FLMA-00006,,CGCON-SIOCON-13992,59.77010,-39.88419,,30,2692,
-AR35-05,,,,GI03FLMA-RIS01-03-DOSTAD000,6,1,2019-08-07T13:34:00,,CGMGI-03FLMA-00006,,CGINS-DOSTAD-00515,59.77010,-39.88419,,30,2692,
-AR35-05,,,,GI03FLMA-RIS01-04-PHSENF000,6,1,2019-08-07T13:34:00,,CGMGI-03FLMA-00006,,CGINS-PHSENF-P0141,59.77010,-39.88419,,30,2692,
-AR35-05,,,,GI03FLMA-RIS01-05-FLORTD000,6,1,2019-08-07T13:34:00,,CGMGI-03FLMA-00006,,CGINS-FLORTD-01212,59.77010,-39.88419,,30,2692,
+AR35-05,,,,GI03FLMA-RI000-00-CTDMOH000,6,1,2019-08-07T13:34:00,,CGMGI-03FLMA-00006,,CGINS-CTDMOH-12218,59.7701,-39.88419,,1689,2692,
+AR35-05,,,,GI03FLMA-RI000-00-CTDMOH100,6,1,2019-08-07T13:34:00,,CGMGI-03FLMA-00006,,CGINS-CTDMOH-12619,59.7701,-39.88419,,2589,2692,
+AR35-05,,,,GI03FLMA-RI000-00-CTDMOH400,6,1,2019-08-07T13:34:00,,CGMGI-03FLMA-00006,,CGINS-CTDMOH-12395,59.7701,-39.88419,,2289,2692,
+AR35-05,,,,GI03FLMA-RI000-00-CTDMOH700,6,1,2019-08-07T13:34:00,,CGMGI-03FLMA-00006,,CGINS-CTDMOH-12387,59.7701,-39.88419,,1989,2692,
+AR35-05,,,,GI03FLMA-RI000-00-VELPTB000,6,1,2019-08-07T13:34:00,,CGMGI-03FLMA-00006,,CGINS-VELPTB-11970,59.7701,-39.88419,,1689,2692,
+AR35-05,,,,GI03FLMA-RI000-00-VELPTB100,6,1,2019-08-07T13:34:00,,CGMGI-03FLMA-00006,,CGINS-VELPTB-12197,59.7701,-39.88419,,2589,2692,
+AR35-05,,,,GI03FLMA-RI000-00-VELPTB400,6,1,2019-08-07T13:34:00,,CGMGI-03FLMA-00006,,CGINS-VELPTB-12095,59.7701,-39.88419,,2289,2692,
+AR35-05,,,,GI03FLMA-RI000-00-VELPTB700,6,1,2019-08-07T13:34:00,,CGMGI-03FLMA-00006,,CGINS-VELPTB-11981,59.7701,-39.88419,,1989,2692,

```

Comment:
The OSNAP depths are incorrect (as is the spreadsheet - my bad) they should:
1689 –> 1692
2589 –> 2592
2289 –> 2292 
1989 –> 1992
So they were right to begin with.

Reply:
corrected